src/Init/Meta.lean

```diff
@@ -211,24 +212,6 @@ def setInfo (info : SourceInfo) : Syntax → Syntax
   | ident _ rawVal val pre => ident info rawVal val pre
   | stx                    => stx
 
-partial def replaceInfo (info : SourceInfo) : Syntax → Syntax
-  | node k args => node k <| args.map (replaceInfo info)
-  | stx         => setInfo info stx
-
-def copyHeadInfo (s : Syntax) (source : Syntax) : Syntax :=
-  match source.getHeadInfo with
-  | none      => s
-  | some info => s.setHeadInfo info
-
-def copyTailInfo (s : Syntax) (source : Syntax) : Syntax :=
-  match source.getTailInfo with
-  | none      => s
-  | some info => s.setTailInfo info
-
-def copyInfo (s : Syntax) (source : Syntax) : Syntax :=
- let s := s.copyHeadInfo source
- s.copyTailInfo source
```

Comment:
@leodemoura I removed these and others just to be sure we're not accidentally using them, but I can re-add them if you want